### PR TITLE
SwimTrack

### PR DESCRIPTION
**Pull request by saumilthecode**
preliminary review :D

### CODE REVIEW

File: submissions/SwimTrack/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**Did you use #arcade to log your progress?**: Yes
+
+**Testflight link**: https://testflight.apple.com/join/ejTWWr5B
+
+**How many users on testflight?**: 17 (so far. MacOS sequla borked testflighting halfway through so i didnt ask poeple to download it)
+
+**Ship URL**: https://hackclub.slack.com/abc (URL of your slack post in the #ship channel on Hack Club Slack) 
+
+it is not complete yet so no ship :p (the onboarding is abit borked rn)
```
Comment (faisalsayed10) [inline submissions/SwimTrack/README.md:28]: would love if u could fix everything, get it completed and ship it :)

File: submissions/SwimTrack/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**Video Demo URL**: [https://youtube.com/xyz](https://www.youtube.com/gxe)
```
Comment (faisalsayed10) [inline submissions/SwimTrack/README.md:30]: would be great if u can attach the demo link

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
left a review in testflight

**Comment by saumilthecode** [comment]
ok so i currently cant figure tipkit out for the life of me but am open to the rest of the app being checked on first so here's the link to me running through it

https://hackclub.slack.com/files/U078K08NN2Y/F07F75Y5XFY/simulator_screen_recording_-_iphone_15_-_2024-08-04_at_09.33.37.mp4

**Comment by saumilthecode** [comment]
wow managed to make the .ds_stores dissapear

**Comment by saumilthecode** [comment]
Hello! @faisalsayed10 i finished up with some final changes and think this is worthy enough of the grant now :)

**Review by faisalsayed10 — APPROVED** [review]
great work on the updates :)

**Pull request merged**